### PR TITLE
fix(qa): authlib 0.14.3 breaks sheepdog

### PR DESCRIPTION
**Pull request by themarcelor**
This version of authlib breaks our Sheepdog service.

### Bug Fixes
This should address the following error:
```
sheepdog-service     |     from authlib.client import OAuthClient
sheepdog-service     |   File "/usr/local/lib/python3.6/site-packages/authlib/client/__init__.py", line 9, in <module>
sheepdog-service     |     from .oauth_client import OAuthClient, OAUTH_CLIENT_PARAMS
sheepdog-service     |   File "/usr/local/lib/python3.6/site-packages/authlib/client/oauth_client.py", line 1, in <module>
sheepdog-service     |     from authlib.integrations._client import RemoteApp as OAuthClient
sheepdog-service     | ModuleNotFoundError: No module named 'authlib.integrations._client'
sheepdog-service     | unable to load 
```

### CODE REVIEW

**Comment by paulineribeyre** [comment]
could you use the "dependency updates" PR description section to describe this change instead of "bug fixes", and not copy stacktraces in PR description sections - they should be bullet points

**Comment by paulineribeyre** [comment]
and i was going to say, don't merge before testing sheepdog with this branch...

**Comment by lepture** [comment]
@themarcelor oh, sorry to break your code. The `authlib.client` module was deprecated, I forgot to make it compatible for v0.14.3.

I'm wondering why are you using this `OAuthClient`. Your importted `OAuthClient` is designed to write framework integrations. Instead, in your case, you are using Flask, A Flask version OAuth is available: https://docs.authlib.org/en/latest/client/flask.html

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ cached-property = "~=1.4"
 cdiserrors = "~=1.0"
 xmltodict = "~=0.9"
 
-authlib = "~=0.11"
+authlib = "<0.14.3"
```
Comment (fantix) [inline pyproject.toml:14]: We should not do this if `authutils` itself is not broken by authlib 0.14.3, or we're building a multi-repo monolith.

**Pull request merged**